bubble_sort.rb

```diff
@@ -0,0 +1,33 @@
+# Implementing Sorting Algorithm
+
+# Warmup: Bubble Sort
+
+def bubble_sort(array_2)
+  loop do
+    swapped = false
+    (0..array_2.length-2).each do |i|
+      if array_2[i] > array_2[i+1]
+        temp = array_2[i]
```

Comment:
To swap 2 elements, `a` and `b` in ruby - `a, b = b, a`.This will eliminate the use of `temp` variable.